modulemd/v2/include/modulemd-2.0/modulemd-component.h

```diff
@@ -110,4 +112,18 @@ modulemd_component_set_rationale (ModulemdComponent *self,
 const gchar *
 modulemd_component_get_rationale (ModulemdComponent *self);
 
+
+/**
+ * modulemd_component_equals:
+ * @self: (not nullable); This #ModulemdComponent.
```

Comment:
```suggestion
 * @self: (not nullable): This #ModulemdComponent.
```

Please make sure to review the results of the automated CI when submitting PRs. This error appears in the failure logs.

Reply:
fixed